Dockerfile-3.1

```diff
@@ -0,0 +1,53 @@
+FROM debian:buster
+LABEL maintainer="Razvan Crainea <razvan@opensips.org>"
+
+USER root
+
+# Set Environment Variables
+ENV DEBIAN_FRONTEND noninteractive
+
+ARG OPENSIPS_VERSION=3.1
+ARG OPENSIPS_BUILD=releases
+
+#install basic components
+RUN apt update -qq && apt install -y gnupg2 ca-certificates
+
+#add keyserver, repository
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 049AD65B
+RUN echo "deb https://apt.opensips.org buster ${OPENSIPS_VERSION}-${OPENSIPS_BUILD}" >/etc/apt/sources.list.d/opensips.list
+
+RUN apt update -qq && apt install -y opensips
+
+ARG OPENSIPS_CLI=false
+RUN if [ ${OPENSIPS_CLI} = true ]; then \
+    echo "deb https://apt.opensips.org buster cli-${OPENSIPS_BUILD}" >/etc/apt/sources.list.d/opensips-cli.list \
+    && apt update -qq && apt -y install opensips-cli \
+    ;fi
+
+ARG OPENSIPS_JSON_MODULE=false
+RUN if [ ${OPENSIPS_JSON_MODULE} = true ]; then \
+    apt-get -y install opensips-json-module \
+    ;fi
+
+ARG OPENSIPS_MYSQL_MODULE=false
+RUN if [ ${OPENSIPS_MYSQL_MODULE} = true ]; then \
+    apt-get -y install opensips-mysql-module \
+    ;fi
+
+ARG OPENSIPS_HTTP_MODULE=false
+RUN if [ ${OPENSIPS_HTTP_MODULE} = true ]; then \
+    apt-get -y install opensips-http-modules \
+    ;fi
+
+ARG OPENSIPS_DIALPLAN_MODULE=false
+RUN if [ ${OPENSIPS_DIALPLAN_MODULE} = true ]; then \
+    apt-get -y install opensips-dialplan-module \
+    ;fi
+
+RUN rm -rf /var/lib/apt/lists/*
+
+EXPOSE 5060/udp
+
+COPY run-3.1.sh /run-3.1.sh
+
+ENTRYPOINT ["/run-3.1.sh"]
```

Comment:
I'm not sure adding a new `Dockerfile` is the right approach, I'd rather create a different branch (3.0), and make the changes on master. I believe this is easier for people to work this out.

Reply:
Dockerfile is completely correct. I have tested it. Creating a 3.0 branch is optional, but not required